cypress/integration/login.spec.js

```diff
@@ -73,7 +73,7 @@ describe('login', () => {
 
     cy.findByLabelText('Email*').type(existingUser.email);
     cy.findByLabelText('Password*').type(existingUser.password);
-    cy.findByText('Submit').click();
+    cy.get('button[type=submit]').click();
```

Comment:
can you instead use the cypress-testing-library methods?

Reply:
Neat. I didn't know this was a thing. I'll try that.